### PR TITLE
Enable Topic Ads on Test Environment

### PR DESCRIPTION
**Pull request by RetiredBlob**
Resolves https://jira.dev.bbc.co.uk/browse/WSTEAMA-65

**Overall change:**
Add the ad leaderboard to topic pages on test

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/pages/TopicPage/TopicPage.jsx [hunk 1]
```diff
@@ -31,6 +36,12 @@ const TopicPage = ({ pageData }) => {
   const { lang, translations } = useContext(ServiceContext);
   const { title, description, promos, pageCount, activePage } = pageData;
 
+  const { enabled: adsEnabled } = useToggle('ads');
+  const { isAmp, showAdsBasedOnLocation } = useContext(RequestContext);
```
Comment (andrewscfc) [inline src/app/pages/TopicPage/TopicPage.jsx]: We don't support amp for topic pages afaik, could we remove the amp checking logic?

**Review by andrewscfc — COMMENTED** [review]
Do we need any tests around showing ads? What have we done on other pages?

**Review by jroebu14 — APPROVED** [review]

**Review by eagerterrier — COMMENTED** [review]
Just a quick note - all branches (or at the very least the PR title) should reference the JIRA ticket number. No need to do so on this one now, but please remember for future.

File: src/app/pages/TopicPage/index.test.jsx [hunk 2]
```diff
@@ -32,14 +51,15 @@ describe('A11y', () => {
...
-        lang="pcm"
-        service="pidgin"
-      />,
-    );
+    const { container, queryByRole } = render(<TopicPageWithContext />);
     expect(queryByRole('list')).toBeInTheDocument();
     expect(container.getElementsByTagName('li').length).toEqual(4);
   });
+
+  it('should show ads when enabled', () => {
```
Comment (andrewscfc) [inline src/app/pages/TopicPage/index.test.jsx:60]: Thanks for adding the tests, we could do with independently checking:
- When ads are enabled but not in a location to show ads
- When ads are disabled, they do not show

**Review by andrewscfc — APPROVED** [review]
Thanks for the tests and removing the amp logic 👍

**Pull request merged**